external/interface/MathUtils.h

```diff
@@ -0,0 +1,38 @@
+#ifndef DELPHYS_EXTERNAL_MATHUTILS_H_
+#define DELPHYS_EXTERNAL_MATHUTILS_H_
+
+#include "TMath.h"
+
+#include <cmath>
+#include <iostream>
+
+namespace delphys {
+
+static const Double_t kPi = TMath::Pi();
+static const Double_t kTwoPi = TMath::TwoPi();
+
+inline Double_t ComputeDeltaPhi(Double_t phi1, Double_t phi2) {
```

Comment:
cant you just use root functions of these?

Reply:
is there root functions?
functions in ROOT::Math::VectorUtils require vector or TLorentzVector